content/docs/reference/types.mdx

```diff
@@ -18,3 +17,8 @@ Tina contains multiple tools and applications designed to improve collaboration
 - [`reference`](/docs/reference/types/reference/)
 - [`object`](/docs/reference/types/object/)
 - [`rich-text`](/docs/reference/types/rich-text/)
+
+<MinimizeText
+  heading="Here we go!"
+  text={<>Test me please Now a second line</>}
+/>
```

Comment:
I dont like the name, should be based off 
https://www.w3schools.com/tags/tag_summary.asp